assets/section-password.css

```diff
@@ -339,5 +339,5 @@ details.modal .modal__toggle-open {
   width: 7rem;
   height: 2rem;
   vertical-align: top;
-  color: var(--color-text);
+  color: rgb(var(--color-base-text));
```

Comment:
This needs to be `--color-forground` instead otherwise the icon isn't updating it's color: ![](https://screenshot.click/21-07-at6fe-i8jdb.png)